### PR TITLE
inisializing react structure by LAABID ABDESSAMAD at 15-03-2024 00:57

### PR DESCRIPTION
**Pull request by aplusInDev**
simple react files structure to begin hr pro project

### CODE REVIEW

**Review by farah-am — APPROVED** [review]
reviewed by farah

**Review by farah-am — CHANGES_REQUESTED** [review]
reviewed by farah 2

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "hrpro",
```
Comment (farah-am) [inline package.json:2]: commented by farah

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```
Comment (farah-am) [inline package.json:12]: commented by farah2

**Review by farah-am — COMMENTED** [review]
comment1

**Comment by aplusInDev** [comment]
done

**Pull request merged**